src/test/kotlin/info/szadkowski/contact/service/mail/MailMessageServiceTest.kt

```diff
@@ -0,0 +1,76 @@
+package info.szadkowski.contact.service.mail
+
+import info.szadkowski.contact.model.MessageRequest
+import info.szadkowski.contact.properties.MailAddressesProperties
+import info.szadkowski.contact.service.MessageService
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.junit.jupiter.MockitoExtension
+import org.springframework.mail.MailException
+import org.springframework.mail.javamail.JavaMailSender
+import javax.mail.Message
+import javax.mail.internet.InternetAddress
+import javax.mail.internet.MimeMessage
+
+@ExtendWith(MockitoExtension::class)
+class MailMessageServiceTest {
+    lateinit var service: MailMessageService
+
+    @Mock
+    lateinit var sender: JavaMailSender
+
+    @Mock
+    lateinit var mimeMessage: MimeMessage
+
+    @BeforeEach
+    fun setUp() {
+        val mailAddressesProperties = MailAddressesProperties()
+        mailAddressesProperties.senderMail = "sender@address.com"
+        mailAddressesProperties.recipientMail = "recipient@address.com"
+        Mockito.`when`(sender.createMimeMessage()).thenReturn(mimeMessage)
```

Comment:
You can check out mockk library. Plays really well with Kotlin.

Reply:
I like it :)